api/wsHub.go

```diff
@@ -34,6 +41,48 @@ var connHandler = func(s *melody.Session) {
 	if err != nil {
 		log.WithError(err).Error("can't write initial stats to session")
 	}
+	var uid uint = 0
+	if tumLiveContext.User != nil {
+		uid = tumLiveContext.User.ID
+	}
+	if tumLiveContext.Course.ChatEnabled {
+		sendServerMessageWithBackoff(s, uid, tumLiveContext.Stream.ID, "Welcome to the chatroom! Please be nice to each other and stay on topic if you want this feature to stay active.", TypeServerInfo)
+	}
+	if !tumLiveContext.Course.AnonymousChatEnabled {
```

Comment:
Not sure if this can happen, but it could prevent some weird behavior in case of errors.
```suggestion
	if !tumLiveContext.Course.AnonymousChatEnabled && tumLiveContext.Course.ChatEnabled {
```